tests/transforms/test_qcut.py

```diff
@@ -1827,10 +1828,12 @@ def circuit(x):
             qml.RZ(0.133, wires=1)
             return qml.expval(qml.PauliZ(wires=[0]))
 
+        spy = mocker.spy(qcut, "qcut_processing_fn")
         x = np.array(0.531, requires_grad=True)
         cut_circuit = qcut.cut_circuit(circuit)
 
         assert round(cut_circuit(x), 8) == round(float(circuit(x)), 8)
+        assert spy.call_count == 1
```

Comment:
Nice! There is also: https://docs.python.org/3/library/unittest.mock.html#unittest.mock.Mock.assert_called_once